lib/index.js

```diff
@@ -57,6 +58,38 @@ function setupTracker(spec) {
   };
 }
 
+function checkDataFiles() {
+  const { ECOBALYSE_DATA_DIR } = process.env;
```

Comment:
How about passing the env as an arg? It could make it easy to unit test, even if it's not mandatory for this specific function.

```js
function checkDataFiles(env = {}) {
  const { ECOBALYSE_DATA_DIR } = env;
```